src/cc/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ include_directories(${LIBELF_INCLUDE_DIRS})
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/libbpf/include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/libbpf/include/uapi)
 add_definitions(${LLVM_DEFINITIONS})
-configure_file(libbcc.pc.in ${CMAKE_CURRENT_BINARY_DIR}/libbcc.pc @ONLY)
+configure_file(libbcc.pc.in libbcc.pc @ONLY)
+configure_file(bcc_version.h.in bcc_version.h @ONLY)
```

Comment:
The above two configure_files do not sound right. The destination file should be ${CMAKE_CURRENT_BINARY_DIR}/libbcc.pc and ${CMAKE_CURRENT_BINARY_DIR}/bcc_version.h, I think.

Reply:
Sorry for the lack of explanation, but that is the default in `configure_file`:

> https://cmake.org/cmake/help/latest/command/configure_file.html

> `<output>`
> Path to the output file or directory. A relative path is treated with respect to the value of CMAKE_CURRENT_BINARY_DIR. If the path names an existing directory the output file is placed in that directory with the same file name as the input file.

Reply:
Thanks. I see.

The buildbot reported build failures at examples/cpp directory,
```
[ 85%] Building CXX object examples/cpp/CMakeFiles/HelloWorld.dir/HelloWorld.cc.o
/tmp/rpmbuild.wYRV5Q/BUILD/bcc/examples/cpp/HelloWorld.cc:11:10: fatal error: bcc/bcc_version.h: No such file or directory
 #include "bcc/bcc_version.h"
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [examples/cpp/CMakeFiles/HelloWorld.dir/build.make:63: examples/cpp/CMakeFiles/HelloWorld.dir/HelloWorld.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:924: examples/cpp/CMakeFiles/HelloWorld.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 85%] Linking CXX executable CPUDistribution
```

May you need to tweak the CMakefile there since the newer bcc_version.h may not make into install directory yet.

Reply:
Right. Fixed in 683f139d by adding bcc_version.h's directory to include dirs.